examples/resources/anxcloud_virtual_server/resource.tf

```diff
@@ -0,0 +1,59 @@
+data "anxcloud_core_location" "anx04" {
+  code = "ANX04"
+}
+
+data "anxcloud_template" "all_templates" {
+  location_id   = data.anxcloud_core_location.anx04.id
+  template_type = "templates"
+}
+
+locals {
+  debian_11_templates = values({
+    for i, template in data.anxcloud_template.all_templates.templates : tostring(i) => template
+    if substr(template.name, 0, 9) == "Debian 11"
+  })
+}
+
+resource "anxcloud_vlan" "example" {
+  count           = 2
+  location_id     = data.anxcloud_core_location.anx04.id
+  vm_provisioning = true
+}
+
+resource "anxcloud_virtual_server" "example" {
+  hostname      = "example-terraform"
+  location_id   = data.anxcloud_core_location.anx04.id
+  template_id   = local.debian_11_templates[0].id
+  template_type = "templates"
+
+  cpus     = 4
+  memory   = 4096
+  password = "flatcar#1234$%"
```

Comment:
can we change the example to using a SSH key? Using a password is not recommended